extensions/gif-search/src/components/GifDetailsActions.tsx

```diff
@@ -37,13 +37,21 @@ export function getActions(item: IGif, showViewDetails: boolean, service?: Servi
 
   const removeFav = () => dispatch({ type: "remove", save: true, favIds: actionIds, service });
 
+  const slugify = (str: string) =>
```

Comment:
I think this logic belongs in the service model's map function. It can then be applied to the slug properly of it which all gif services already use.